src/main/java/org/jboss/modules/Main.java

```diff
@@ -370,50 +378,51 @@ public static void main(String[] args) throws Throwable {
         ModularURLStreamHandlerProvider.addHandlerModule(module);
         ModularContentHandlerFactory.addHandlerModule(module);
 
-        // at this point, having a security manager already installed will prevent correct operation.
-
-        final SecurityManager existingSecMgr = System.getSecurityManager();
-        if (existingSecMgr != null) {
-            System.err.println("An existing security manager was detected.  You must use the -secmgr switch to start with a security manager.");
-            System.exit(1);
-            return; // not reached
-        }
+        if (SECURITY_MANAGER_SUPPORTED) {
```

Comment:
I think it would be marginally clearer to use the `actualVersion` variable above, instead of a constant field which is only going to be used within this method but which persists on the class forever.